modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/SubscriptionStore.java

```diff
@@ -109,17 +135,13 @@ public SubscriptionStore() throws AndesException {
         if (isTopic) {
             // In topic scenario if this is a durable topic it's in cluster queue subscription map,
             // hence we need to check both maps
-
-            // Topic map is only have topic messages
-            subscriptionList.addAll(getSubscriptionsInMap(destination,
-                    clusterTopicSubscriptionMap, SUBSCRIPTION_TYPE.ALL));
-
-            // Get durable topic subscriptions from Queue map
-            subscriptionList.addAll(getSubscriptionsInMap(destination,
-                    clusterQueueSubscriptionMap, SUBSCRIPTION_TYPE.TOPIC_SUBSCRIPTION));
+            if (isBitmap)
```

Comment:
its better to use curly braces to define condition scopes all the time. (helps reading)